src/sql/workbench/parts/connection/browser/connection.contribution.ts

```diff
@@ -55,6 +61,48 @@ actionRegistry.registerWorkbenchAction(
 	AddServerAction.LABEL
 );
 
+CommandsRegistry.registerCommand('azdata.connect',
+	function (accessor, args) {
```

Comment:
Can you type the args here? What exactly are you expecting the shape of this to be?